test/mainnet/buffer.spec.sol

```diff
@@ -11,8 +11,9 @@ import {AssertUtils} from "test/utils/AssertUtils.sol";
 import {MockERC4626} from "test/mainnet/mocks/MockERC4626.sol";
 import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import {MockProvider} from "test/unit/mocks/MockProvider.sol";
+import {BaseRules} from "script/rules/BaseRules.sol";
 
-contract VaultMainnetInvariantsTest is Test, AssertUtils, MainnetActors {
+contract VaultMainnetInvariantsTest is Test, AssertUtils, MainnetActors, BaseRules {
```

Comment:
the changes to the rest of the tests are refactor related changes for the rules.